kafka_utils/kafka_rolling_restart/main.py

```diff
@@ -70,6 +70,13 @@ def parse_opts():
         '-c',
         help='cluster name, e.g. "uswest1-devc" (defaults to local cluster)',
     )
+    parser.add_argument(
+        '--broker-ids',
+        '-b',
+        required=False,
+        type=str,
```

Comment:
Have you considered using `argparse`'s `nargs` mechanism to directly have a list of integers here?
Using a comma-separated list of brokers is fine, but using built-in tooling could be a good idea

Context: https://docs.python.org/2/library/argparse.html#example

Reply:
sure I will take a look.